pyflask/manageNeuroconv/manage_neuroconv.py

```diff
@@ -109,6 +121,39 @@ def coerce_schema_compliance_recursive(obj, schema):
     )
 
 
+def autocomplete_format_string(info: dict) -> str:
+    from neuroconv.tools.path_expansion import construct_path_template
+
+    base_directory = info["base_directory"]
+    filesystem_entry_path = info["path"]
+
+    if not is_path_contained(filesystem_entry_path, base_directory):
+        raise "Path is not contained in the provided base directory."
```

Comment:
```suggestion
        raise ValueError("Path is not contained in the provided base directory.")
```

Always attach an Error class type with a message

For generic things you can always do `assert condition: bool, message: str` as shorthand, which raises an `AssertionError` type

So looks like this logic has to account for symlinks